File: ternary operator.c
import java.io.*;
import java.util.Scanner;
class ternary {
	 public static void main(String[] args) { 
	     Scanner input = new Scanner(System.in);
	     
	     System.out.println("Enter age: ");
	     int age = input.nextInt();
	     
	    String vote = (age>=18) ? "eligible to vote" : "not eligible to vote";
	     
	     System.out.println(vote);
	 }
}

File: ternary operator.java
import java.util.Scanner;
class even{
    public static void main(String[] args){
        Scanner inp = new Scanner(System.in);
        int n = inp.nextInt();
        System.out.println((n % 2 == 0) ? "Even" : "Odd");
    }
}

or
/*
import java.util.Scanner;
class even{
    public static void main(String[] args){
        Scanner inp = new Scanner(System.in);
        int n = inp.nextInt();
        String ans;
        ans = (n % 2 == 0) ? "Even" : "Odd";
        
        System.out.println(ans);
    }
}
*/
